EIPS/eip-5114.md

```diff
@@ -0,0 +1,75 @@
+	function ownerOf(uint256 index) external view returns (address nftAddress, uint256 nftTokenId);
+	
+	// returns a censorship resistant URI with details about this token collection
+	// the metadata returned by this is merged with the metadata return by `tokenUri(uint256)`
+	// the tokenUri **MUST** be immutable and content addressable (e.g., ipfs://)
+	// the tokenUri **MUST NOT** point at mutable/censorable content (e.g., https://)
+	// data from `tokenUri` takes precedence over data returned by this method
+	// any external links referenced by the content at `tokenUri` also **MUST** follow all of the above rules
+	function collectionUri() external view returns (string tokenUri);
+	
+	// returns a censorship resistant URI with details about this token instance
+	// the tokenUri **MUST** be immutable and content addressable (e.g., ipfs://)
+	// the tokenUri **MUST NOT** point at mutable/censorable content (e.g., https://)
+	// data from this takes precedence over data returned by `collectionUri`
+	// any external links referenced by the content at `tokenUri` also **MUST** follow all of the above rules
+	function tokenUri(uint256 tokenId) external view returns (string tokenUri);
+}
+```
+
+## Rationale
+### Immutability
+By requiring that tokens can never move, we both guarantee non-separability and non-mergeability among collections of soulbound tokens that are bound to a single NFT while simultaneously allowing users to aggressively cache results.
+### Content Addressable URIs Required
+Soulbound tokens are meant to be permanent badges/indicators attached to a persona.
+This means that not only can the user not transfer ownership, but the minter also cannot withdraw/transfer/change ownership as well.
+This includes mutating or removing any remote content as a means of censoring or manipulating specific users.
+### No Specification for tokenUri Data Format
+The format of the data pointed to by `collectionUri()` and `tokenUri(uint256)` is intentionally left out of this standard in favor of separate standards that can be iterated on in the future.
+The immutability constraints are the only thing defined by this to ensure that the spirit of this token is maintained, regardless of the specifics of the data format.
+
+## Backwards Compatibility
+This is a new token type and is not meant to be backward compatible with any existing tokens other than existing viable souls (like EIP-721 tokens).
+
+## Security Considerations
+Users of tokens that claim to implement this EIP must be diligent in verifying they actually do.
+A token author can create a token that, upon initial probing of the API surface, may appear to follow the rules when in reality it doesn't.
+For example, the contract could allow transfers via some mechanism and simply not utilize them initially.
+
+It should also be made clear that Soulbound tokens are not bound to a human, they are bound to a persona.
```

Comment:
Should check for consistent capitalization of Soulbound

Reply:
```suggestion
It should also be made clear that soulbound tokens are not bound to a human, they are bound to a persona.
```